src/Aspire.Hosting.Azure/Bicep/search.bicep

```
@@ -54,21 +56,38 @@ resource search 'Microsoft.Search/searchServices@2020-08-01' = {
     replicaCount: replicaCount
     partitionCount: partitionCount
     hostingMode: hostingMode
+    disableLocalAuth: true
   }
   tags: tags
 }
 
-var primaryKey = search.listAdminKeys(search.apiVersion).primaryKey
-var searchServiceName = search.name
+// Find list of roles and GUIDs in https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles
```

```suggestion
// Find list of roles and GUIDs in https://learn.microsoft.com/azure/role-based-access-control/built-in-roles
```

People get mad when we have locales in our links.